whylog/tests/test_constraints/__init__.py

```diff
@@ -0,0 +1,55 @@
+from unittest import TestCase
+
+from whylog.constraints import IdenticalConstraint, TimeConstraint
+from whylog.constraints.exceptions import ConstructorGroupsError, ConstructorParamsError
+
+
+class TestIdenticalConstraint(TestCase):
+    def test_constructor_success(self):
+        IdenticalConstraint(dict(), [(0, 1), (2, 1), (2, 4)])
```

Comment:
Missing test for convert_to_user_constraint_intent. For me this is the most important method of this class. Add it here.